### PR TITLE
Code formatting

### PR DESCRIPTION
**Pull request by inyono**
This PR adds code formatters for most of the languages used in the projects (except Twig since I didn't find a good code formatter).

Please code review

- [x] https://github.com/serlo-org/athene2/pull/785/commits/42bf904142dc1365b3af2ece43d9039307c9d266
- [x] https://github.com/serlo-org/athene2/pull/785/commits/fac3427f57051786d1ba2f2097847a9314a29984

and scan https://github.com/serlo-org/athene2/pull/785/commits/b708849ce2aa21125fa852bf772507ab0d9cc862 to check if you are okay with coding changes introduced by php-cs-fixer.

### Added

- Add `package.json` so that we can use `yarn` as task runner
- Add yarn script `start` that executes `docker-compose up`
- Add yarn script `format:prettier` that executes prettier (formats Markdown, YAML, JSON, etc.)
- Add yarn script `format:php` that executes php-cs-fixer (formats PHP according to PSR-2 with some additional rules)

## TODO

- [x] Update changelog
- [x] Add `yarn lint` that checks if files are formatted according to prettier / php-cs-fixer config
- [x] Execute `yarn lint` in CI

### CODE REVIEW

File: src/module/Common/src/Common/Filter/Shortify.php [hunk 1]
```diff
@@ -45,16 +45,17 @@ class Shortify implements FilterInterface
...
             'wann', 'warum', 'was', 'weiter', 'weitere', 'wenn', 'wer', 'werde',
             'werden', 'werdet', 'weshalb', 'wie', 'wieder', 'wieso', 'wir',
-            'wird', 'wirst', 'wo', 'woher', 'wohin', 'zu', 'zum', 'zur', 'über'
-        ]
+            'wird', 'wirst', 'wo', 'woher', 'wohin', 'zu', 'zum', 'zur', 'über',
+        ],
     ];
 
-    static $regex = null;
+    public static $regex = null;
```
Comment (Knorrke) [inline src/module/Common/src/Common/Filter/Shortify.php]: this probably should be private.

File: src/module/Common/src/Common/Filter/Shortify.php [hunk 2]
```diff
@@ -18,7 +18,7 @@ class Shortify implements FilterInterface
      *
      * Please use the smallest possible bundle for a language
      */
-    static $stopWords = [
+    public static $stopWords = [
```
Comment (Knorrke) [inline src/module/Common/src/Common/Filter/Shortify.php]: this probably should be private.

File: HOWTOS.md [hunk 3]
```diff
@@ -126,41 +127,39 @@ crontab -e
...
+1. `$1`
+1. array
+1. `array\(([a-zA-Z\'\=\>\(\)\-]+)\)`
+1. `{ $1 }`
+1. array =>
+1. `\=\>`
+1. `:`
+1. translate
+1. `translate\((.*)\)`
+1. `$1 | trans`
```
Comment (Knorrke) [inline HOWTOS.md]: this markdown is not correct, we have to check this manually

**Comment by inyono** [comment]
@Knorrke Apart from the CI / lint stuff, this is done now

**Pull request merged**